packages/@ember/-internals/metal/lib/property_events.ts

```diff
@@ -31,6 +31,7 @@ let deferred = 0;
   @param {String} keyName The property key (or path) that will change.
   @param {Meta} meta The objects meta.
   @return {void}
+  @since 3.1.0
```

Comment:
This marks this available _since_ `3.1.0` but this commit (and the one exposing the export from `@ember/object` is not in a tag till `v3.8.0-beta.1`. How does that work?